setup.py

```diff
@@ -39,6 +39,13 @@ def get_requirements():
     keywords="aws-encryption-sdk aws kms encryption",
     license="Apache License 2.0",
     install_requires=get_requirements(),
+    # pylint: disable=fixme
+    # TODO: Point at MPL main branch once Python MPL is merged into main.
+    extras_require={
+        "MPL": ["aws-cryptographic-material-providers @" \
+                "git+https://github.com/aws/aws-cryptographic-material-providers-library.git@" \
+                "lucmcdon/python-mpl#subdirectory=AwsCryptographicMaterialProviders/runtimes/python"],
```

Comment:
This should point to a specific version yes?

Reply:
Yeah... I'll update the comment here

Reply:
Done